packages/tools/client-debugger/client-debugger-chrome-extension/package.json

```diff
@@ -13,13 +13,13 @@
 	"author": "Microsoft and contributors",
 	"scripts": {
 		"build": "npm run build:genver && concurrently npm:build:compile npm:lint",
-		"build:compile": "concurrently npm:build:tsc npm:build:copy-resources && npm run webpack",
+		"build:compile": "concurrently npm:tsc npm:build:webpack",
 		"build:copy-resources": "copyfiles -u 1 \"src/**/*.{css,json,html}\" dist",
 		"build:full": "npm run build",
 		"build:full:compile": "npm run build:compile",
 		"build:genver": "gen-version",
 		"build:tsc": "npm run tsc",
```

Comment:
Yeah this is one that's a complete "no-op" in that it's just a restructuring of the same commands to make fluid-build happy. As a general statement I'm not saying that I always agree with the fluid-build warnings and that this PR is just me trying to make it happy so that we get fewer warnings 😆